apps/ledger-live-mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -30,8 +30,9 @@
     <uses-permission android:name="android.permission.RECORD_AUDIO" /> <!-- Needed by Live apps -->
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" /> <!-- Required for the webview camera/audio media -->
     <uses-sdk
+        android:minSdkVersion="21"
         android:targetSdkVersion="27"
-        tools:overrideLibrary="com.ledger.reactnative" />
+        tools:overrideLibrary="com.ledger.reactnative, ca.jaysoo.extradimensions" />
```

Comment:
what is this? is it really needed?
apparently the package that refers to it is deprecated and no longer needed: https://github.com/Sunhat/react-native-extra-dimensions-android?tab=readme-ov-file#notice

Reply:
I think that it's not needed anymore, but I don't want to make the removal of this lib on this PR as it includes too much stuff already. I think that this must be done on a dedicated one and we may have other libraries concerned too.
It's needed to override the SDK declaration that fails without it because minSDK declared in the lib is not supported anymore.